NumLib/DOF/DOFTableUtil.cpp

```diff
@@ -30,4 +44,60 @@ std::vector<GlobalIndexType> getIndices(
     return indices;
 }
 
+double norm(GlobalVector const& x, unsigned const global_component,
+            MathLib::VecNormType norm_type,
+            LocalToGlobalIndexMap const& dof_table, MeshLib::Mesh const& mesh)
+{
+    // TODO that also includes ghost nodes.
+    std::size_t const n_nodes = mesh.getNumberOfNodes();
+
+    // TODO for PETSc some global accumulation is necessary.
+    switch(norm_type)
+    {
+    case MathLib::VecNormType::NORM1:
```

Comment:
Why not different function for each norm type? What is the advantage to decide the norm type via parameter?